wallet-new/test/MarshallingSpec.hs

```diff
@@ -30,8 +30,8 @@ import qualified Cardano.Wallet.Util as Util
 
 -- | Tests whether or not some instances (JSON, Bi, etc) roundtrips.
 spec :: Spec
-spec = describe "Marshalling & Unmarshalling" $ do
-    describe "Roundtrips" $ do
+spec = parallel $ describe "Marshalling & Unmarshalling" $ do
+    parallel $ describe "Roundtrips" $ do
```

Comment:
As we are at it, should we write an `aesonRoundtripProp` for `TransactionStatus`? As we are manipulating freeform-strings, it looks like a good thing to do 😉

Reply:
Good call! I wish there was a way to say "For any non-derived instance of ToJSON/FromJSON, ensure that they round trip."